persistence-modules/java-mongodb/src/test/java/com/baeldung/bsontojson/JSONDateFormatEpochTime.java

```diff
@@ -0,0 +1,18 @@
+package com.baeldung.bsontojson;
+
+import org.bson.json.Converter;
+import org.bson.json.StrictJsonWriter;
+
+/**
+ * Convertor to epoch time
+ */
+public class JSONDateFormatEpochTime implements Converter<Long> {
```

Comment:
let's add the suffix "Converter" here

Reply:
btw converting value to String, shouldn't this be the default behaviour?

Reply:
Because I saw when it serialize for that type of filed is converted into the LocalDateTime style.

Reply:
Is that a problem with our articles? I mean my intention is that we should have as less custom code as possible unless needed by the article.

Reply:
If we want to keep the same output in the test we should create that converter. That is because of this: 
https://github.com/mongodb/specifications/blob/master/source/extended-json.rst#datetime